content/subtleties.md

```diff
@@ -14,37 +14,39 @@ Also see Pierre Weis's [Frequently asked Questions about Caml](http://caml.inria
 * **Semicolons and `if` statements**: TODO (I can fill this
   in later unless someone else is inspired to do it first. -mars0i)
 
-* **Weak types**: What is "the value restriction"?
+* **Weak type variables**: What is "the value restriction"?
   Why do some of my type variables start with underscore?
   What is the difference between a function with (for example) type `'a -> 'b`,
   and one with type `'_weak1 -> '_weak2`?  Why am I getting a
   compiler error like these? (It worked when I tried it out in utop!)
```

Comment:
It would be useful to get an actual example here of some code that causes this warning to trigger.

Reply:
it is not a warning, but yes, I will add an example, why it breaks type soundness and segfaults the program.